### PR TITLE
Oauth2 beta partdeux

### PR DESCRIPTION
**Pull request by mystiberry-okta**
Replaces this pull request because of spuriously added commits: https://github.com/okta/okta.github.io/pull/248/commits

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
Ready for review.

File: _source/docs/api/resources/oidc.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-redirect_from: "docs/api/rest/oidc.html"
 ---
 
 # Overview
 
 The OpenID Connect API endpoints enable clients to use [OIDC workflows](http://openid.net/specs/openid-connect-core-1_0.html) with Okta.
 
-This API also enables you to use Okta as a broker for social-login scenarios where the user authenticates against identity providers like Google, Facebook, LinkedIn, and Microsoft,
-and returns an Okta session.
+With OpenID Connect and Okta, a client can use Okta as a broker when the user authenticates against identity providers like Google, Facebook, LinkedIn, and Microsoft,
```
Comment (yuliu-okta) [inline _source/docs/api/resources/oidc.md]: With OpenID Connect, a client can ....

File: _source/docs/api/resources/oidc.md [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
+and the client returns an Okta session.
```
Comment (yuliu-okta) [inline _source/docs/api/resources/oidc.md]: the client gets an Okta session

**Comment by royalchan-okta** [comment]
Looks good! 

I just realized that the OAuth 2.0 Introspection Request should also mention ID tokens as well. Might be good to change 

> The API takes an Access Token or Refresh Token and returns whether it is active or not. If the token is active, additional data about the token will also be returned. If the token is invalid, expired, or revoked, it will be considered inactive. An implicit client can only introspect its own tokens, while a confidential client may inspect all access tokens.

to

> The API takes an Access Token, Refresh Token, or an [ID Token](oidc.html#id-token) and returns whether it is active or not. If the token is active, additional data about the token will also be returned. If the token is invalid, expired, or revoked, it will be considered inactive. An implicit client can only introspect its own tokens, while a confidential client may inspect all access tokens.

and then change the OIDC ID Token line

> ID Tokens should always be validated by the client to ensure their integrity.

to mention the introspection endpoint.

**Comment by mystiberry-okta** [comment]
@royalchan-okta  Added your changes.

File: _source/docs/api/resources/oidc.md [hunk 3]
```diff
@@ -19,6 +19,7 @@ which is a [JSON web token (JWT)](https://tools.ietf.org/html/rfc7519) that cont
 as well as claims about the authenticated user.
 
 ID Tokens should always be [validated](#validating-id-tokens) by the client to ensure their integrity.
+You can use an [OAuth 2.0 introspection request](/docs/api/resources/oauth2.html#introspection-request) for validation.
```
Comment (royalchan-okta) [inline _source/docs/api/resources/oidc.md]: maybe add "or manually as follows" to make it obvious the id token is a JWT and can be manually verified

**Comment by royalchan-okta** [comment]
I don't see the oauth change to add the id token @mystiberry-okta, minor comment on the other change.

**Comment by mystiberry-okta** [comment]
Yah, I nuked the change in the other file. Restoring now with your other comment.

**Comment by mystiberry-okta** [comment]
@royalchan-okta I think this is what we want! I'm testing links now, if any are broken I'll fix them.

**Comment by mystiberry-okta** [comment]
Shippable, @royalchan-okta ? Thanks!

**Comment by royalchan-okta** [comment]
🚢 looks good!

**Pull request merged**